NuGet.master.config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
```

Comment:
Not sure, it's part of the build template.
https://github.com/aspnet/Universe/tree/dev/build-template